packages/mobile/features/configure_workout/ConfigureWorkout.tsx

```diff
@@ -1,21 +1,54 @@
 import React, { useState } from 'react';
-import { View, TextInput } from 'react-native';
+import { View, TextInput, Button, StyleSheet } from 'react-native';
 import i18nString, { Messages } from '@common/i18n';
+import { CreateWorkoutScreenProps } from '@mobile/features/configure_workout/CreateWorkoutScreen';
+import useBoundStore from '@mobile/stores/useBoundStore';
 
-function ConfigureWorkout() {
+const styles = StyleSheet.create({
+  row: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+  },
+  input: {
+    height: 40,
```

Comment:
hard coding height for an input prolly not good? doesn't scale well with accessibility settings (ppl changing their font size etc)